app/views/DR/HomeScreen/DashboardCards.js

```diff
@@ -0,0 +1,52 @@
+import { Card, Text } from 'native-base';
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import Colors from '../../../constants/colors';
+import styles from './style';
+
+export default function dashboardCards({
+  navigation,
+  t,
+  confirmed,
+  deaths,
+  recovered,
+  current,
+}) {
+  const dashboardCategories = [
+    { label: 'label.positive_label', styles: '', value: confirmed },
+    {
+      label: 'label.deceased_label',
+      styles: { color: Colors.BUTTON_LIGHT_TEX },
+      value: deaths,
+    },
+    {
+      label: 'label.recovered_label',
+      styles: { color: Colors.GREEN },
+      value: recovered,
+    },
+    {
+      label: 'label.case_day_label',
+      styles: { color: Colors.SUN },
+      value: current,
+    },
+  ];
+  const CONFIRMED_COVID_CASES_URL =
+    'https://coronavirusrd.gob.do/casos-de-covid-19-confirmados/';
+
+  return dashboardCategories.map((obj, index) => (
+    <TouchableOpacity
+      style={styles.shadowBorder}
+      key={index}
```

Comment:
Is not a best practice to use indexes as key in react list, you can use the label instead
```suggestion
      key={obj.label}
```